infrastructure/sentry-subgraph/schema/events/PoolFactory.graphql

```diff
@@ -39,3 +39,15 @@ type PoolFactoryConfig @entity(immutable: false) {
   unstakeEsXaiDelayPeriod: BigInt!
   updateRewardBreakdownDelayPeriod: BigInt!
 }
+
+type PoolChallenges @entity(immutable: false) {
```

Comment:
The entity name should always be singular, the graph will automaticall allow quering for lists by calling entity(s)

If the entity is called `PoolChallenges` we will not be able to query for mutliple PoolChallenges in one query.